### PR TITLE
fix: update auth table and dependencies

### PR DESCRIPTION
**Pull request by soridalac**
### What does this PR do?
update new auth table format
### What issues does this PR fix or reference?
@W-16736180@

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,8 +10,8 @@
     "@oclif/core": "^4",
     "@salesforce/core": "^8.6.2",
     "@salesforce/kit": "^3.2.3",
-    "@salesforce/plugin-info": "^3.4.12",
-    "@salesforce/sf-plugins-core": "^11.3.10",
+    "@salesforce/plugin-info": "^3.4.9",
```
Comment (WillieRuemmele) [inline package.json:13]: why is this dependency going backwards? 12->9?

**Review by WillieRuemmele — APPROVED** [review]

**Comment by WillieRuemmele** [comment]
### QA Notes 
---
✅ : using new table
⚠️ : removed history, I ran the new command locally and saw the table, but if I scrolled up, I only scrolled to the top of the table, no other commands output was visible

File: src/commands/org/list/auth.ts [hunk 2]
```diff
@@ -38,16 +38,19 @@ export default class ListAuth extends SfCommand<AuthListResults> {
...
+      this.table({
+        data: mappedAuths,
+        columns: [
+          { key: 'alias', name: 'ALIAS' },
+          { key: 'username', name: 'USERNAME' },
+          { key: 'orgId', name: 'ORG ID' },
+          { key: 'instanceUrl', name: 'AUTH METHOD' },
+          { key: 'oauthMethod', name: 'AUTH METHOD' },
+          hasErrors ? { key: 'error', name: 'ERROR' } : 'error',
+        ],
```
Comment (mdonnalley) [inline src/commands/org/list/auth.ts:52]: add 

`title: 'authenticated orgs'` 

and remove `this.styledHeader('authenticated orgs')` just above this

File: src/commands/org/list/auth.ts [hunk 3]
```diff
@@ -38,16 +38,19 @@ export default class ListAuth extends SfCommand<AuthListResults> {
...
       this.styledHeader('authenticated orgs');
-      this.table(mappedAuths, columns);
+
+      this.table({
+        data: mappedAuths,
+        columns: [
+          { key: 'alias', name: 'ALIAS' },
+          { key: 'username', name: 'USERNAME' },
+          { key: 'orgId', name: 'ORG ID' },
+          { key: 'instanceUrl', name: 'AUTH METHOD' },
```
Comment (mdonnalley) [inline src/commands/org/list/auth.ts:49]: should be `{ key: 'instanceUrl', name: 'INSTANCE URL' }`

File: src/commands/org/list/auth.ts [hunk 4]
```diff
@@ -38,16 +38,19 @@ export default class ListAuth extends SfCommand<AuthListResults> {
...
+          { key: 'oauthMethod', name: 'AUTH METHOD' },
+          hasErrors ? { key: 'error', name: 'ERROR' } : 'error',
```
Comment (mdonnalley) [inline src/commands/org/list/auth.ts:51]: `error` shouldn't be in the `columns` array if `hasErrors` is false

**Pull request merged**